src/controller.ts

```diff
@@ -33,22 +33,30 @@ export class TestExplorerStatusBarController implements TestController {
 
     const testAdapterState = new TestAdapterState();
     this.testAdapterStates.push(testAdapterState);
+
+    let timeoutID:NodeJS.Timeout;
```

Comment:
This should have a space
```suggestion
    let timeoutID: NodeJS.Timeout;
```